### PR TITLE
Fix spelling of Tasseled, so consistent. And multiple changes to the data description. The same changes have been made to CMI.

### PR DESCRIPTION
**Pull request by vnewey**

### CODE REVIEW

**Review by pindge — APPROVED** [review]
Good

File: dev/services/wms/ows/ows_cfg.py [hunk 1]
```diff
@@ -5303,33 +5303,35 @@ def swap_scale(new_scale: list, style: dict):
...
+Geoscience Australia Landsat Collection 2 Tasseled Cap Wetness Percentiles 1986-2018, 25 metre, 100km tile, Australian Albers Equal Area projection (EPSG:3577). 
+Data is only visible at higher resolutions; when zoomed-out the available area will be displayed as a shaded region.
 
-The ability to map partial inundation is also crucial to understand patterns of human water use. We need to be able to identify potential wetlands and groundwater dependent ecosystems on the Australian continent so that they can be monitored and managed.
+Areas that are partially covered in water, or where water is mixed with vegetation when viewed from above, provide habitat for a wide range of aquatic organisms. 
 
-The tasseled cap wetness percentiles provide a multi-decadal summary of landscape wetness that can be used to identify wetlands and groundwater ecosystems.
+The ability to map partial inundation is also crucial to understand patterns of human water use. We need to be able to identify potential wetlands and groundwater dependent ecosystems on the Australian continent so that they can be monitored and managed
```
Comment (pindge) [inline dev/services/wms/ows/ows_cfg.py:5311]: was there suppose to be a fullstop here? @vnewey

**Review by pindge — CHANGES_REQUESTED** [review]

File: dev/services/wms/ows/ows_cfg.py [hunk 2]
```diff
@@ -5303,33 +5303,35 @@ def swap_scale(new_scale: list, style: dict):
...
 
 They are intended for use as inputs into classification algorithms to identify potential wetlands and groundwater dependent ecosystems, and characterise salt flats, clay pans, salt lakes and coastal land forms.
 
 This product provides valuable discrimination for characterising:
 
-vegetated wetlands
-salt flats
-salt lakes
-coastal land cover classes
+vegetated wetlands, salt flats, salt lakes, coastal land cover classes
```
Comment (pindge) [inline dev/services/wms/ows/ows_cfg.py:5321]: if not dot points? should they be inline with the above line?

Comment (vnewey) [reply dev/services/wms/ows/ows_cfg.py:5321]: The dot points all appeared on the same line.  Do you know how to make them appear as dot points?

Comment (pindge) [reply dev/services/wms/ows/ows_cfg.py:5321]:
```

 - vegetated wetlands,
 - salt flats,
 - salt lakes,
 - coastal land cover classes
```

image

**Review by pindge — APPROVED** [review]

**Pull request merged**